spec/aruba/api_spec.rb

```diff
@@ -1071,6 +1071,48 @@ def actual_permissions
 
         expect(ENV[variable]).to eq '1'
       end
+
```

Comment:
Forget about my comment extracting the env thing. I just saw I already created a PR for that.